### PR TITLE
docs/providers: aws_rds_cluster - Provides correct values for max and min capacity with PostgreSQL

### PR DESCRIPTION
**Pull request by lmammino**
This PR updates the `Resource: aws_rds_cluster` documentation page with the correct list of accepted values for `min_capacity ` and `max_capacity ` when using PostgreSQL.

From the AWS docs: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.setting-capacity.html

Screenshot 2021-02-10 at 16 43 54

Props to @pegaslee and @templed for finding this one out!

### CODE REVIEW

File: website/docs/r/rds_cluster.html.markdown [hunk 1]
```diff
@@ -209,8 +209,8 @@ resource "aws_rds_cluster" "example" {
 ```
 
 * `auto_pause` - (Optional) Whether to enable automatic pause. A DB cluster can be paused only when it's idle (it has no connections). If a DB cluster is paused for more than seven days, the DB cluster might be backed up with a snapshot. In this case, the DB cluster is restored when there is a request to connect to it. Defaults to `true`.
-* `max_capacity` - (Optional) The maximum capacity. The maximum capacity must be greater than or equal to the minimum capacity. Valid capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, and `256`. Defaults to `16`.
-* `min_capacity` - (Optional) The minimum capacity. The minimum capacity must be lesser than or equal to the maximum capacity. Valid capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, and `256`. Defaults to `1`.
+* `max_capacity` - (Optional) The maximum capacity. The maximum capacity must be greater than or equal to the minimum capacity. Valid capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, and `256` (or `1`, `2`, `4`, `8`, `16`, `32`, `64`, `192`, and `384` for PostgreSQL). Defaults to `16`.
```
Comment (breathingdust) [inline website/docs/r/rds_cluster.html.markdown:212]: Thanks for this @lmammino, looking at the docs though it looks like 1 is not a valid capacity value for PostgreSQL and its likely better we are explicit that it applies to Aurora.

```suggestion
* `max_capacity` - (Optional) The maximum capacity for an Aurora DB cluster in `serverless` DB engine mode. The maximum capacity must be greater than or equal to the minimum capacity. Valid Aurora MySQL capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, `256`. Valid Aurora PostgreSQL capacity values are (`2`, `4`, `8`, `16`, `32`, `64`, `192`, and `384`). Defaults to `16`.
```

File: website/docs/r/rds_cluster.html.markdown [hunk 2]
```diff
@@ -209,8 +209,8 @@ resource "aws_rds_cluster" "example" {
...
+* `min_capacity` - (Optional) The minimum capacity. The minimum capacity must be lesser than or equal to the maximum capacity. Valid capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, and `256` (or `1`, `2`, `4`, `8`, `16`, `32`, `64`, `192`, and `384` for PostgreSQL). Defaults to `1`.
```
Comment (breathingdust) [inline website/docs/r/rds_cluster.html.markdown:213]:
```suggestion
* `min_capacity` - (Optional) The minimum capacity for an Aurora DB cluster in `serverless` DB engine mode. The minimum capacity must be lesser than or equal to the maximum capacity. Valid Aurora MySQL capacity values are `1`, `2`, `4`, `8`, `16`, `32`, `64`, `128`, `256`. Valid Aurora PostgreSQL capacity values are (`2`, `4`, `8`, `16`, `32`, `64`, `192`, and `384`). Defaults to `1`.
```

**Review by breathingdust — CHANGES_REQUESTED** [review]

**Comment by breathingdust** [comment]
LGTM 🚀 Thanks @lmammino!

**Comment by lmammino** [comment]
Happy that this was useful! Thanks for the great review, @breathingdust !

When is this going to be relesed?

**Pull request merged**